init.lua

```diff
@@ -10,6 +10,11 @@ Licensed under the zlib license. See LICENSE.md for more information.
 
 moreores = {}
 
+local default_tin = false
+if minetest.registered_items["default:tin_ingot"] then
+	default_tin = true
+end
+
```

Comment:
`local default_tin = minetest.registered_items["default:tin_ingot"] ~= nil` would be shorter.